pages/apis/apis.md.erb

```diff
@@ -0,0 +1,50 @@
+# API Overview
+
+## Authentication
+
+The Buildkite RESET and GraphQL APIs expect the token to be provided via the `Authorization` HTTP header:
+
+```bash
+curl -H "Authorization: Bearer $TOKEN" https://api.buildkite.com/v2/user
+```
+
+All Webhooks containg the `X-Buildkite-Token` which allows you to verify the authenticity of the request.
+
+```yaml
+Example: 309c9c842d8565adec5d7469160059f9
+```
+
+Generate an [Access Token](https://buildkite.com/user/api-access-tokens).
+
+## REST API
+
+The Buildkite REST API aims to give you complete programmatic access and control of Buildkite to extend, integrate and automate anything to suit your particular needs.
+
+1. Generate an [API Access Token](https://buildkite.com/user/api-access-tokens) with as much [scope](/docs/apis/managing-api-tokens#token-scopes) as you need.
+2. Make requests to https://api.buildkite.com using the token you generated in the `Authorization` header:
+
+    ```bash
+    curl -H "Authorization: Bearer $TOKEN" https://api.buildkite.com/v2/user
+    ```
+
+More information about the [REST API](/docs/apis/rest-api).
+
+## GraphQL
+
+The Buildkite GraphQL API provides an alternative to the REST API.
+It allows for more efficient retrieval of data by enabling you to fetch multiple, nested resources in a single request.
+The Buildkite GraphQL API is currently in beta.
+
+1. Generate an [API Access Token](https://buildkite.com/user/api-access-tokens) with `GraphQL` scope
+2. Open the [GraphQL Explorer](https://graphql.buildkite.com/explorer)
+3. Follow the [GraphQL Tutorial](https://building.buildkite.com/tutorial-getting-started-with-graphql-queries-and-mutations-11211dfe5d64)
+
+More information about the [GraphQL API](/docs/apis/graphql-api).
+
+## Webhooks
+
+Webhooks allow you to monitor and respond to events within your Buildkite organization, providing a real time view of activity and allowing you to extend and integrate Buildkite into your systems.
+
+Webhooks can be added and configured on your [organization's Notification Services settings](https://buildkite.com/organizations/-/services) page.
```

Comment:
This link is also present in `webhooks.md.erb` and doesn't work. What should it be instead?